charts/nd-common/templates/_datadog.tpl

```diff
@@ -28,6 +28,24 @@ ad.datadoghq.com/{{ .Chart.Name }}.instances: |-
   ]
 {{- end }}
 
+{{/*
+This is datalog logging configuration. We take the .Values.scrapeLogs and
+.Values.scrapeLogsProcessingRules map and convert into list of objects converted
+into json supported by datadog config. If source and service tag values not provided
+we add default values to it.
+*/}}
+
```

Comment:
Remove this newline, OR replace line 36 with `*/ -}}` to wipe out this newline..